manifest.toml

```diff
@@ -56,7 +56,8 @@ ram.runtime = "50M"
     main.default = 9494
 
     [resources.apt]
+    packages = "curl"
```

Comment:
Why this package?

Thank you for your work!

Reply:
Trying to solve: 
https://ci-apps-dev.yunohost.org/ci/job/7276

```
    - Error validating manifest using schema: in key resources > apt
       {'extras': {'yarn': {'repo': 'deb https://dl.yarnpkg.com/debian/ stable main', 'key': 'https://dl.yarnpkg.com/debian/pubkey.gpg', 'packages': ['yarn']}}} is not valid under any of the given schemas
```

Reply:
Oh I guess it's something that could be updated in https://github.com/YunoHost/apps/blob/master/schemas/manifest.v2.schema.json (I think it's the correct file) and making this valid?

Reply:
> Oh I guess it's something that could be updated in https://github.com/YunoHost/apps/blob/master/schemas/manifest.v2.schema.json (I think it's the correct file) and making this valid?

I really don't, just find that workaround, as curl is already included in YunoHost

Reply:
I don't see what could be wrong with it